app/uk/gov/hmrc/nisp/views/formatting/NispMoney.scala

```diff
@@ -17,8 +17,31 @@
 package uk.gov.hmrc.nisp.views.formatting
 
 import play.twirl.api.Html
-import uk.gov.hmrc.play.views.formatting.Money
+
+import scala.math.BigDecimal.RoundingMode._
 
 object NispMoney {
-  def pounds(value: BigDecimal): Html = Money.pounds(value, if (value.isValidInt) 0 else 2)
+  def pounds(
```

Comment:
FYI. This is what we have in tax summaries.

For useability we have:
https://github.com/hmrc/tax-summaries-frontend/blob/main/app/views/includes/CurrencyAmountView.scala.html
I think if I remember well it switch the position of the minus and pound sign for reading. So it read "minus 300 pounds" but it displays -£300

Reply:
I just lifted and shifted the functionality from Play UI so it was a like for like change, so I could get rid of that dependency. I will raise a ticket to use play-frontend-hmrc currency UI, if it exists.

Reply:
A currency element in play-frontend-hmrc would be wonderful. Ping me if you find anything :)